unbound/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
@joewilliams Just want to touch base on this. Have you had a chance to look at Derek's feedback?